src/ControlForm.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { setParksToShow } from './actions';
+
+const ControlForm = ({ usStates, parks, parksToDisplay, setParksToShow, visitedParkCodes, bucketListParkCodes }) => {
+  const showAllParks = (event) => {
+    event.preventDefault();
+    setParksToShow('all');
+  }
+
+  const showVisitedParks = (event) => {
+    event.preventDefault();
+    setParksToShow('visitedParks');
+  }
+
+  const showBucketList = (event) => {
+    event.preventDefault();
+    setParksToShow('bucketList');
+  }
+  
+  const getState = (event) => {
+    setParksToShow(event.target.value);
+    // this.setMapToState(event.target.value, this.props.usStates[event.target.value]);
+  }
+
+  // setMapToState = (stateName, stateObj) => {
+  //   this.props.setMapToState(stateName, stateObj);
+  // }
+
+  return (
+    <div className="filter-controls">
+      <div className="filters">
+        <button onClick={showAllParks} id="show-all-button">Show All Parks</button>
+        <button onClick={showVisitedParks} id="show-visited-button"><img className="loc-icons" src="./assets/marker-icon-green.png" alt="green icon" /> Show {visitedParkCodes.length} Visited Parks</button>
+        <button onClick={showBucketList} id="show-bucket-button"><img className="loc-icons" src="./assets/marker-icon-violet.png" alt="purple icon" /> Show {bucketListParkCodes.length} Bucket List Parks</button>
+      </div>
+      {/* <label className="filter-label">View Specific Park:
+        <select id="select-menu" value={this.props.stateName} onChange={this.getState}>
+          <option value="default">Please pick a state</option>
+          {
+            Object.keys(this.props.usStates).map(usState => {
+              return (<option value={[usState]} key={this.props.usStates[usState].abbreviation}>{[usState]}</option>)
+            })
+          }
+        </select>
+      </label> */}
+      <label className="filter-label">View Parks By State:
+        <select id="select-menu" value={parksToDisplay} onChange={getState}>
+          <option value="default">Please pick a state</option>
+        {
+          Object.keys(usStates).map(usState => {
+            return(<option value={[usState]} key={usState.abbreviation}>{[usState]}</option>)
+          })
+        }
+        </select>
+      </label>
```

Comment:
I think this is where we used to show the US State component, when a user chose a state. Is that something we still want to show?

Reply:
I took that out for now - I left the component in there but took out the render. This was something the mod-2 instructors mentioned in our eval, they aren't sure if it adds much value to the app, which I kind of agree. I feel like we were mainly including it because the project spec required us to work with two datasets. Happy to talk about this more, I also had some other ideas for things to add (like the search by park name) which may also make the app cluttered if we still have the state card displaying.